### PR TITLE
Add missing Rabbitmq configuration for Rummager

### PR DESCRIPTION
**Pull request by bishboria**
The following changes were missing/in the wrong place in order for RabbitMQ and Rummager to work together.

### CODE REVIEW

File: modules/govuk/manifests/apps/rummager.pp [hunk 1]
```diff
@@ -98,10 +98,6 @@
     process_type   => 'publishing-api-document-indexer',
   }
 
-  govuk_rabbitmq::monitor_consumers {'rummager-publishing-api-document-indexer':
```
Comment (rboulton) [inline modules/govuk/manifests/apps/rummager.pp]: I think we still want to monitor the consumer.  Was this meant to have been moved into the `rummager/rabbitmq.pp` file?

File: hieradata/class/rabbitmq.yaml [hunk 2]
```diff
@@ -8,6 +8,7 @@ govuk::node::s_rabbitmq::apps_using_rabbitmq:
   - panopticon
   - publishing_api
   - stagecraft
+  - rummager
```
Comment (alexmuller) [inline hieradata/class/rabbitmq.yaml]: Nitpick, this needs to be bumped up one line to be alphabetical :)

**Comment by rboulton** [comment]
Aren't there some test settings that need to be added too?  Or are those not actually needed?

**Comment by alexmuller** [comment]
There was a bit of trouble with adding the test exchange because `govuk::apps::publishing_api::rabbitmq` also defines an exchange called `published_documents_test@/` and Puppet doesn't like duplicate resources named the same thing.

**Comment by rboulton** [comment]
:+1: please merge if you're happy.

**Comment by rboulton** [comment]
Actually, I know you're happy, so I'm merging myself.

**Pull request merged**